addons/website_slides/security/ir.model.access.csv

```diff
@@ -23,7 +23,7 @@ access_slide_embed_all,slide.embed.all,model_slide_embed,,1,0,0,0
 access_slide_embed_user,slide.embed.user,model_slide_embed,base.group_user,1,1,1,1
 access_slide_slide_link_all,slide.slide.link.all,model_slide_slide_link,,1,0,0,0
 access_slide_slide_link_officer,slide.slide.link.officer,model_slide_slide_link,website_slides.group_website_slides_officer,1,1,1,1
-access_slide_slide_resource_all,slide.slide.resource.all,model_slide_slide_resource,,1,0,0,0
-access_slide_slide_resource_public,slide.slide.resource.public,model_slide_slide_resource,base.group_public,0,0,0,0
+access_slide_slide_resource_all,slide.slide.resource.all,model_slide_slide_resource,,0,0,0,0
```

Comment:
Does that change something ? As this changes master data in stable and as final result seems to be the same, is it required ?

Reply:
Unless I missed something, this change should ensure that visitors do not have access to the slide resources.
As it is, they have read access granted by `access_slide_slide_resource_all`.

Reply:
Additional note: for stable purpose, I would keep access_slide_slide_resource_public and remove in master (as removing data is always a bit meh in stable).

Reply:
Additional additional group: you also have to give rights for base.user group (otherwise portal has more rights than standard users).